cmd/cli/cmd/alpha/manifest-gen/implementation/helm.go

```diff
@@ -62,3 +64,24 @@ func NewHelm() *cobra.Command {
 
 	return cmd
 }
+
+func generateHelmManifests(opts common.ManifestGenOptions) (manifestgen.ManifestCollection, error) {
+	helmchartInfo, err := askForHelmChartDetails()
+	if err != nil {
+		return nil, errors.Wrap(err, "while asking for Helm chart details")
+	}
+
+	var helmCfg manifestgen.HelmConfig
+	helmCfg.ManifestPath = common.CreateManifestPath(common.ImplementationManifest, opts.ManifestPath)
+	helmCfg.ChartName = helmchartInfo.Name
+	helmCfg.ManifestMetadata = opts.Metadata
+	helmCfg.ChartRepoURL = helmchartInfo.Repo
+	helmCfg.ChartVersion = helmchartInfo.Version
+	helmCfg.InterfacePathWithRevision = opts.InterfacePath
+
+	files, err := manifestgen.GenerateHelmManifests(&helmCfg)
```

Comment:
As discussed:
- In Helm generator we already support both local Helm chart and remote one
- We will have another question: "Select Helm chart location" with enum values: "local", "remote"
- In case of remote: get Helm chart name, version and url 
- In case of local: get path
- Add TODO for handling getting Helm parameters in loop (see comment above)

Reply:
Implemented, now we can choose between "remote" and "local". :+1: